### PR TITLE
Fix App bugs

### PR DESCRIPTION
**Pull request by akbarjorayev**

### CODE REVIEW

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

File: src/components/Footer.js [hunk 1]
```diff
@@ -13,7 +13,16 @@ function Footer() {
...
         <Container>
           <Grid container style={{ display: 'flex', alignItems: 'center' }}>
-            <Grid item xs={6} align="left" style={{ whiteSpace: 'nowrap' }}>
+            <Grid
+              item
+              xs={6}
+              align="left"
+              style={{
+                whiteSpace: 'nowrap',
+                overflow: 'hidden',
```
[the remark this answers is not in the document]
Comment (akbarjorayev) [reply src/components/Footer.js:22]: oldn qanaqa edi bomasa commit suggestion jonat

File: src/components/Navbar.js [hunk 2]
```diff
@@ -81,7 +82,7 @@ function Navbar() {
     >
       <Container>
         <Grid container>
-          <Grid item xs={4}>
+          <Grid item xs={4} className="avatar_con">
```
[the remark this answers is not in the document]
Comment (akbarjorayev) [reply src/components/Navbar.js:85]: css bilan center qlsh uchun

File: src/components/NavbarSimple.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ function NavbarSimple() {
     >
       <Container>
         <Grid container>
-          <Grid item xs={4} align="left">
+          <Grid item xs={4} align="left" justifyContent='center'>
```
[the remark this answers is not in the document]
Comment (akbarjorayev) [reply src/components/NavbarSimple.js:18]: oldn qanaqa edi bomasa commit suggestion jonat

**Review by akbarjorayev — COMMENTED** [review]
**LGTM**

File: src/pages/css/Company.css [hunk 4]
```diff
@@ -1,5 +1,9 @@
 @import url('./Profile.css');
 
+.company_con h1 {
```
Comment (nazarovctrl) [inline src/pages/css/Company.css:3]: bu nega kere?

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

File: src/pages/css/Company.css (hunk 4 above)
Comment (akbarjorayev) [reply src/pages/css/Company.css:3]: telegramdan yozganman

**Review by akbarjorayev — COMMENTED** [review]
Good

**Comment by akbarjorayev** [comment]
Is `Footer.js` okay? If so, it's ready to **MERGE**

File: src/components/Footer.js [hunk 5]
```diff
@@ -13,14 +13,20 @@ function Footer() {
       <Box p={1}>
         <Container>
           <Grid container style={{ display: 'flex', alignItems: 'center' }}>
-            <Grid item xs={6} align="left" style={{ whiteSpace: 'nowrap' }}>
-              <Typography variant="body2" color="text.secondary">
+            <Grid item xs={6} align="left">
```
Comment (nazarovctrl) [inline src/components/Footer.js:16]: Bu joyga nega teydin orqaga qaytar

File: src/pages/Profile.js [hunk 6]
```diff
@@ -200,7 +200,7 @@ function Profile() {
     const parent = img.parentElement;
 
     const imgMaxSize = (await IMG.getData(fileInput.current.files[0])).size.max;
-    const parentMinSize = IMG.minSize(parent.clientWidth, parent.clientHeight);
+    const parentMinSize = IMG.maxSize(parent.clientWidth, parent.clientHeight);
```
Comment (nazarovctrl) [inline src/pages/Profile.js:203]: parentMaxSize db nomlash keremasmi ?

File: src/pages/css/App.css [hunk 7]
```diff
@@ -1,3 +1,14 @@
+*,
```
Comment (nazarovctrl) [inline src/pages/css/App.css:1]: Bu joyga nega tegdin, orqaga qaytar

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

File: src/pages/Profile.js (hunk 6 above)
Comment (akbarjorayev) [reply src/pages/Profile.js:203]: Togri 🙂‍↕️

File: src/pages/css/App.css (hunk 7 above)
Comment (akbarjorayev) [reply src/pages/css/App.css:1]: Bu narsa kere
For more info: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing)

File: src/components/Footer.js (hunk 5 above)
Comment (akbarjorayev) [reply src/components/Footer.js:16]: Ozn ettin tg ga kir

Comment (akbarjorayev) [reply src/components/Footer.js:16]: > Bu joyga nega teydin orqaga qaytar


Yoqmasa commit suggestion jonat

File: src/pages/css/App.css (hunk 7 above)
Comment (nazarovctrl) [reply src/pages/css/App.css:1]: Bu bugda yogidiyu. Bug bomagani nega tiyvosan

Comment (akbarjorayev) [reply src/pages/css/App.css:1]: Bu yaxshilash uchun

**Review by akbarjorayev — COMMENTED** [review]
**Good!**

**Review by nazarovctrl — APPROVED** [review]

**Pull request merged**